web_sdk/web_engine_tester/lib/golden_tester.dart

```diff
@@ -64,7 +64,7 @@ Future<void> matchGoldenFile(String filename,
             'height': region.height
           },
     'pixelComparison': pixelComparison.toString(),
-    'isCanvaskitTest': useCanvasKit,
+    'isCanvaskitTest': renderer.rendererTag == 'canvaskit',
```

Comment:
I'm curious why you decided to use `renderTag` to check instead of using something like `renderer is CanvaskitRenderer`.

Reply:
The issue is that these particular unit tests are operating on the transformed (post `sdk_rewriter`) version of the engine, and `CanvasKitRenderer` itself is actually private to the engine, and I'd prefer to keep it that way, so I figured looking at the renderer tag is a perfectly fine way to do this, since this is just metadata about the test that's being bundled.

Reply:
That makes sense! It would be great if you could add this explanation as a comment above this line of code.

Reply:
Okay, will do!